### PR TITLE
Add details to adlist table

### PR DESCRIPTION
**Pull request by DL6ER**
**By submitting this pull request, I confirm the following:** 

- [X] I have read and understood the [contributors guide](https://github.com/pi-hole/pi-hole/blob/master/CONTRIBUTING.md), as well as this entire template.
- [X] I have made only one major change in my proposed changes.
- [X] I have commented my proposed changes within the code.
- [X] I have tested my proposed changes, and have included unit tests where possible.
- [X] I am willing to help maintain this change if there are issues with it later.
- [X] I give this submission freely and claim no ownership.
- [X] It is compatible with the [EUPL 1.2 license](https://opensource.org/licenses/EUPL-1.1)
- [X] I have squashed any insignificant commits. ([`git rebase`](http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html))

---
**What does this PR aim to accomplish?:**

Add more details to adlist table, exemplary screenshots:

Screenshot from 2020-12-31 11-23-40

Local lists do not get a link as they are not available for distant clients:
Screenshot from 2020-12-31 11-23-50

**How does this PR accomplish the above?:**

Parse new data in the `gravity.db:adlist` table stored there by https://github.com/pi-hole/pi-hole/pull/3951

**What documentation changes (if any) are needed to support this PR?:**

None

### CODE REVIEW

**Comment by DL6ER** [comment]
Rebased on latest `devel`

**Comment by DL6ER** [comment]
Core PR has already been reviewed and merged.

**Comment by PromoFaux** [comment]
image

Presumably a default date because it has never downloaded since gravity was created..?

**Review by PromoFaux — DISMISSED** [review]

File: scripts/pi-hole/js/groups-adlists.js [hunk 1]
```diff
@@ -30,6 +30,83 @@ $(function () {
...
+    '</td></tr><tr class="dataTables-child"><td>This list was added to Pi-hole&nbsp;&nbsp;</td><td>' +
+    utils.datetimeRelative(data.date_added) +
+    "&nbsp;(" +
+    utils.datetime(data.date_added, false) +
+    ')</td></tr><tr class="dataTables-child"><td>Database entry was last modified&nbsp;&nbsp;</td><td>' +
+    utils.datetimeRelative(data.date_modified) +
+    "&nbsp;(" +
+    utils.datetime(data.date_modified, false) +
+    ')</td></tr><tr class="dataTables-child"><td>The list contents were last updated&nbsp;&nbsp;</td><td>' +
+    (data.date_updated !== null
```
Comment (DL6ER) [inline scripts/pi-hole/js/groups-adlists.js]:
```suggestion
    (data.date_updated > 0
```
should avoid the 1970 date.

**Review by PromoFaux — APPROVED** [review]

**Comment by yubiuser** [comment]
I see a small glitch:
Bildschirmfoto zu 2021-03-19 11-48-07

`https://mirror1.malwaredomains.com/files/justdomains` is not availabe and got `status=4`


`{"data":[{"id":2,"address":"https:\/\/mirror1.malwaredomains.com\/files\/justdomains","enabled":0,"date_added":1579547647,"date_modified":1615746537,"comment":"Pihole default list","date_updated":null,"number":0,"invalid_domains":0,"status":4,"groups":[0]}`

Shouldn't the fa-times (cross symbole) be used?
https://github.com/pi-hole/AdminLTE/blob/187e9ddae6182b156cb3dbc910a3468e2792f5c9/scripts/pi-hole/js/groups-adlists.js#L154

**Pull request merged**